V/Valhalla/bundled/patches/liblz4.pc

```diff
@@ -0,0 +1,10 @@
+prefix=${pcfiledir}/../..
+exec_prefix=${pcfiledir}/../..
+libdir=/workspace/destdir/lib
+includedir=/workspace/destdir/include
```

Comment:
Why not making these relative to prefix?

Reply:
```suggestion
libdir=${prefix}/lib
includedir=${prefix}/include
```